Build/GetWix.ps1

```diff
@@ -0,0 +1,18 @@
+function GetWix {
+    param($target)
+
+    Write-Output "Downloading Wix to $target"
+
+    $file = [IO.Path]::GetTempFileName()

```

Comment:
Doesn't matter - I changed to ZipFile which doesn't require the extension to be .zip.